packages/next/src/builders/build/build.impl.ts

```diff
@@ -27,11 +25,5 @@ function run(
   context: BuilderContext
 ): Observable<BuilderOutput> {
   const root = path.resolve(context.workspaceRoot, options.root);
-  const config = prepareConfig(
-    context.workspaceRoot,
-    options.root,
-    options.outputPath,
-    PHASE_PRODUCTION_BUILD
-  );
-  return from(build(root, config as any)).pipe(map(() => ({ success: true })));
+  return from(build(root)).pipe(map(() => ({ success: true })));
```

Comment:
Do we not need to pass in the config anymore?

Reply:
Not anymore, because Next.js will detect and load the `next.config.js` file automatically.